### PR TITLE
AVI Marketplace Role: to accept license terms and conditions

### PR DESCRIPTION
**Pull request by rahulroshan96**

### CODE REVIEW

**Comment by ramkumar-sankar** [comment]
@abhinavmodi this role needs to be applied at subscription level - not at resource group.  so we have to keep it separate.

File: azure/roles/avi_marketplace_role.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "Name": "AVI Marketplace Role",
+  "IsCustom": true,
+  "Description": "Role provides read/write permissions to accept AVI marketplace license terms and conditions",
```
Comment (ramkumar-sankar) [inline azure/roles/avi_marketplace_role.json]: to "auto accept"

File: azure/roles/avi_marketplace_role.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "Name": "AVI Marketplace Role",
```
Comment (ramkumar-sankar) [inline azure/roles/avi_marketplace_role.json]: AVI Controller Market Place Role

File: azure/roles/avi_marketplace_role.json (hunk 1 above)
Comment (rahulroshan96) [reply azure/roles/avi_marketplace_role.json]: Done

File: azure/roles/avi_marketplace_role.json (hunk 2 above)
Comment (rahulroshan96) [reply azure/roles/avi_marketplace_role.json]: Done

File: azure/roles/avi_marketplace_role.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
-  "Name": "AVI Marketplace Role",
+  "Name": "AVI Controller Market Place Role",
```
Comment (abhinavmodi) [inline azure/roles/avi_marketplace_role.json]: Please change "AVI" to "Avi" wherever used.

File: azure/roles/avi_marketplace_role.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
+  "Name": "Avi Controller Market Place Role",
+  "IsCustom": true,
+  "Description": "Role provides read/write permissions to auto accept Avi marketplace license terms and conditions",
```
Comment (ramkumar-sankar) [inline azure/roles/avi_marketplace_role.json]: sorry one more. can you pls add a comment that this role needs to be applied at the subscription level ?

**Pull request merged**